products/securityscanner/api.yaml

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Product
+name: SecurityScanner
+display_name: Cloud Security Scanner
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://websecurityscanner.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Web Security Scanner API
+    url: https://console.cloud.google.com/apis/library/websecurityscanner.googleapis.com/
```

Comment:
This should actually be the link to the API documentation, rather than the link to enable the API

Reply:
It looks like we use it as a link to enable the API.

Such as:
```apis_required:
  - !ruby/object:Api::Product::ApiReference
    name: Compute Engine API
    url: https://console.cloud.google.com/apis/library/compute.googleapis.com/```

Reply:
oh neat, that's new! I was thinking of the ReferenceLinks section, which it looks like is missing here.